test/math/FloatPoint2DTest.ooc

```diff
@@ -49,9 +50,8 @@ FloatPoint2DTest: class extends Fixture {
 		this add("casting", func {
 			value := "10.00, 20.00"
 			expect(this point3 toString(), is equal to(value))
-//			FIXME: Equals interface
-			expect((FloatPoint2D parse(value)) x, is equal to((this point3) x))
-			expect((FloatPoint2D parse(value)) y, is equal to((this point3) y))
+			expect((FloatPoint2D parse(t"10.00,20.00")) x, is equal to((this point3) x))
```

Comment:
> For now it works as it did before.

Wait, no, it doesn't. It used to be able to parse `10.00, 20.00`, now it no longer is, it can only parse `10.00,20.00`. #643 needs to be fixed before I can merge this, or we will break lots of things.